common/reviews/api/heft-config-file.api.md

```diff
@@ -12,7 +12,15 @@ export class ConfigurationFile<TConfigurationFile> {
     getObjectSourceFilePath<TObject extends object>(obj: TObject): string | undefined;
     getPropertyOriginalValue<TParentProperty extends object, TValue>(options: IOriginalValueOptions<TParentProperty>): TValue;
     // (undocumented)
-    loadConfigurationFileAsync(configurationFilePath: string): Promise<TConfigurationFile>;
+    loadConfigurationFileAsync(configurationFilePath: string, options?: {}): Promise<TConfigurationFile>;
+    // (undocumented)
+    loadConfigurationFileAsync(configurationFilePath: string, options?: {
+        ignoreIfNotExist: false;
+    }): Promise<TConfigurationFile>;
+    // (undocumented)
+    loadConfigurationFileAsync(configurationFilePath: string, options: {
+        ignoreIfNotExist: true;
```

Comment:
This is not a good API design.  If we need a special case with a different return value, make it `tryLoadConfigurationFileAsync()`.

However I question whether we need that.  I'll ask about it offline.